### PR TITLE
PRD-5391 , SP-1749 - Empty cells overwrite cells in the first section pr...

### PR DESCRIPTION
**Pull request by tmorgner**
...inted on the sheet as row-offset is not applied in background cells.

(cherry picked from commit 578cd54)

### CODE REVIEW

**Comment by tmorgner** [comment]
@mdamour1976 , @dkincade Please review, merge.

File: engine/core/test/org/pentaho/reporting/engine/classic/core/bugs/Prd5391.java [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+ *  You should have received a copy of the GNU Lesser General Public License along with this
+ *  program; if not, you can obtain a copy at http://www.gnu.org/licenses/old-licenses/lgpl-2.1.html
+ *  or from the Free Software Foundation, Inc.,
+ *  51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
+ *
+ *  This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY;
+ *  without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ *  See the GNU Lesser General Public License for more details.
+ *
+ *  Copyright (c) 2006 - 2009 Pentaho Corporation..  All rights reserved.
```
Comment (dkincade) [inline engine/core/test/org/pentaho/reporting/engine/classic/core/bugs/Prd5391.java]: copywrite date is incorrect

**Comment by dkincade** [comment]
This looks good to me with the exception of the copywrite header...

**Comment by dkincade** [comment]
Merge is approved

**Pull request closed without merging**